haec/src/horse/controllers/horse.controller.ts

```diff
@@ -3,15 +3,27 @@ import { HorseService } from '../services/horse.service';
 
 @Controller('horses')
 export class HorseController {
-  constructor(private readonly horseService: HorseService) {}
+  constructor(private readonly horseService: HorseService) { }
 
   @Get()
   public async findAll() {
     return this.horseService.getHorses();
   }
 
-  @Post()
-  handlePost(@Body() body: any): string {
-    return this.horseService.writeJSONBody(body);
+  @Post('/create')
+  async create(@Body() body: { showName: string; name: string; feed: string }) {
```

Comment:
I'd preface these with `public`
```suggestion
  public async create(@Body() body: { showName: string; name: string; feed: string }) {
```